pkg/utils/utils.go

```diff
@@ -165,13 +165,15 @@ func ParseLines(lines []string, params map[string]interface{}, logger logrus.Fie
 			key := strings.TrimSpace(parts[0])
 			value := strings.TrimSpace(parts[1])
 			// Skip the quotes in the value if present
-			unquotedValue, err := strconv.Unquote(string(value))
-			if err != nil {
-				logger.WithError(err).Debugf("Failed to unquote value %v for key %v. Just store the original value string", value, key)
-				params[key] = string(value)
-				continue
+			var unquotedValue string
+			// Check if value is double-quoted
+			if strings.Contains(value, `"`) {
```

Comment:
So are we sure the password can never have a `"` in it?

Reply:
Trim will removing leading and trailing `"`. I will need to confirm whether or not a password can begin or end with a double-quote.

Reply:
New approach will deprecate this function.